### DISCUSSION TITLE
Cant login to rustdesk

### DISCUSSION THREAD

**Discussion by happnatious1** [rustdesk/rustdesk#5337] [Q&A] [answered]
Setup a lxc debian 12 container in proxmox and followed the install instructions for debian install. Set to domain and enabled web server. going to rustdesk.mydomain.com:8000 brings up a login prompt but nothing I've tried works. root/mypassword, admin/admin, admin/test1234....
Some googling led me to a redit post saying that this hasn't been implemented yet. How are people using this if you cant login?

**Comment by dinger1986** [comment, marked as answer]
That's the techahold install which uses gohttpserver the username password was given at the end of the install for you to login.

This is only to login to get preconfigured configs
